MafiaCore/Game.cs

```diff
@@ -117,6 +119,19 @@ public void ExecuteVotingPhase()
             ChangeGameState();
         }
 
+        private void FillRolesToAssign()
+        {
+            int numTotalPlayers = PlayerMapping.Count;
+            // If less than 8 players, just have 1 mafia and 1 doctor
+            if (numTotalPlayers < 8)

```

Comment:
Perhaps use this logic? http://www.topped-with-meat.com/connector/mafia.html

Reply:
We're gonna keep track of who is winning more, so for now, my new change should do the trick